stdlib/mexpr/pprint.mc

```diff
@@ -150,16 +154,41 @@ lang FunPrettyPrint = FunAst
 end
 
 lang RecordPrettyPrint = RecordAst
+  sem record2tuple =
+  | TmRecord t ->
+    match t.bindings with [] then
+      None ()
+    else
+      let keys = assocKeys {eq=eqstr} t.bindings in
+      match all stringIsInt keys with false then None () else
+      let intKeys = map string2int keys in
+      let sortedKeys = sort subi intKeys in
+      -- Check if keys are a sequence 0..(n-1)
+      match and (eqi 0 (head sortedKeys))
+                (eqi (subi (length intKeys) 1) (last sortedKeys)) with true then
+        Some (map (lam key. assocLookupOrElse {eq=eqstr} (lam _. never)
```

Comment:
I believe `never` is intended for cases that the compiler can see statically (through the type system/exhaustiveness checking most likely) will never happen, so this will probably be a compile time error once that is implemented.

Reply:
Thanks, changed it into an `error`.